pkg/engine/jmespath/arithmetic.go

```diff
@@ -51,18 +51,12 @@ func ParseArithemticOperands(arguments []interface{}, operator string) (Operand,
 	return op[0], op[1], nil
 }
 
-// Quantity +|- Quantity          -> Quantity
-// Quantity +|- Duration|Scalar   -> error
-// Duration +|- Duration          -> Duration
-// Duration +|- Quantity|Scalar   -> error
-// Scalar   +|- Scalar            -> Scalar
-// Scalar   +|- Quantity|Duration -> error
 
 func (op1 Quantity) Add(op2 interface{}) (interface{}, error) {
 	switch v := op2.(type) {
 	case Quantity:
 		op1.Quantity.Add(v.Quantity)
-		return op1.String(), nil
+		 return op1.String(),nil
```

Comment:
@VedRatan please make sure this file is not changed at all (it shouldn't show in the diff)

Reply:
sure @eddycharly.